core/src/mindustry/ui/fragments/PlayerListFragment.java

```diff
@@ -78,7 +78,7 @@ public void rebuild(){
             NetConnection connection = user.con;
 
             if(connection == null && net.server() && !user.isLocal) return;
-            if(sField.getText().length() > 0 && !user.name.toLowerCase().contains(sField.getText().toLowerCase())) return;
+            if(sField.getText().length() > 0 && !user.name.toLowerCase().replaceAll("\\[(.*?)\\]", "").contains(sField.getText().toLowerCase())) return;
```

Comment:
You should check if their name contains the string, **or** if their name without colors contains the string as well. Someone may want to search for tags.
For colors, use `Strings.stripColors(name)`, as it may be more reliable in the future.